### ISSUE TITLE
Is Hogzilla still alive and kickin'?

### ISSUE THREAD

**Issue by SebJansen** [pauloangelo/hogzilla#9]

**Comment by ppartarr** [comment]
No, the open-source project is dead as far as I can tell... Info from the website shows that the project was bought and only the entreprise version is maintained [1].

> New maintainer: From October/2019, the Hogzilla IDS will be maintained and supported by GuardianKey Cybersecurity. This partnership will bring fresh breath to Hogzilla IDS, enabling advances and improvements in the tool. GuardianKey will keep Hogzilla IDS as open-source and will offer an enterprise version of Hogzilla, which will be officially supported. More info about GuardianKey Cybersecurity in https://guardiankey.io.

[1] https://ids-hogzilla.org
